README.md

```diff
@@ -4,8 +4,14 @@ This is a terminal emulator. Why yet another? Because.
 
 Installation
 ------------
-To install it you will need [rustup.rs](https://rustup.rs), then you can
-install it with Cargo.
+To install it you will need [rustup.rs](https://rustup.rs).
+
+```shell
+curl https://sh.rustup.rs -sSf | sh
+rustup install nightly
```

Comment:
Could be replaced by using

    $ curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain nightly